### DISCUSSION TITLE
Multi Node support

### DISCUSSION THREAD

**Discussion by wodka** [cloudamqp/lavinmq#507] [Ideas]
Hi, is there a plan to support Multi Node installations?

It would be awesome to be able to run lavinmq in kuberentes ie with multiple pods in a stateful set.

**Comment by snichme** [comment]
LavinMQ 2.0 is now released with clustering support: https://lavinmq.com/blog/lavinmq-20 
We have even released 2.0.1 with some bug fixes.
